### PR TITLE
Moving surges in traffic to fundamentals

### PR DESCRIPTION
**Pull request by jason-cf**

### CODE REVIEW

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+pcx_content_type: how-to
+title: Preparing for surges or spikes in web traffic
+---
+
+# Preparing for surges or spikes in web traffic
+
+## Use Cloudflare Cache features to optimize caching
+
+By default, Cloudflare [caches static content](/cache/concepts/default-cache-behavior/) such as images, CSS and JavaScript; however, you can extend Cloudflare caching to work with HTML by creating custom [Cache Rules](/cache/how-to/cache-rules/).
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
By default, Cloudflare [caches static content](/cache/concepts/default-cache-behavior/) such as images, CSS, and JavaScript; however, you can extend Cloudflare caching to work with HTML by creating custom [Cache Rules](/cache/how-to/cache-rules/).
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+### Cache more requests
+
+1. Log in to your Cloudflare dashboard.
+
+2. Go to **Cache** > **Cache Rules** and select **Create Rule**.
+
+3. For `When incoming requests match…`, enter either your entire website or a specific path on your application, based on the **Hostname** or **URI Path**. Refer to the [available fields here](/cache/how-to/cache-rules/settings/#fields)
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
3. For **When incoming requests match**, enter either your entire website or a specific path on your application, based on the **Hostname** or **URI Path**. Refer to the [available fields](/cache/how-to/cache-rules/settings/#fields).
```
UI elements should be **bolded** and we shouldn't include the ellipsis (`...`) if it exists.

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+pcx_content_type: how-to
+title: Preparing for surges or spikes in web traffic
+---
+
+# Preparing for surges or spikes in web traffic
```
Comment (marciocloudflare) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
# Prepare for surges or spikes in web traffic
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+### Cache more requests
+
+1. Log in to your Cloudflare dashboard.
+
+2. Go to **Cache** > **Cache Rules** and select **Create Rule**.
+
+3. For `When incoming requests match…`, enter either your entire website or a specific path on your application, based on the **Hostname** or **URI Path**. Refer to the [available fields here](/cache/how-to/cache-rules/settings/#fields)
+
+4. For `Cache eligibility`, define how these requests should be cached and for how long. Refer to [available cache eligibility settings here](/cache/how-to/cache-rules/settings/#eligible-for-cache-settings)
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
4. For **Cache eligibility**, define how these requests should be cached and for how long. Refer to the available [cache eligibility settings](/cache/how-to/cache-rules/settings/#eligible-for-cache-settings).
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md (hunk 1 above)
Comment (marciocloudflare) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
By default, Cloudflare [caches static content](/cache/concepts/default-cache-behavior/) such as images, CSS and JavaScript. However, you can extend Cloudflare caching to work with HTML by creating custom [Cache Rules](/cache/how-to/cache-rules/).
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+5. You can then monitor the effectiveness of your cache settings using [Cache Analytics](/cache/performance-review/cache-analytics/) and update according to our [Cache performance guide](/cache/performance-review/cache-performance/).
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
5. You can then monitor the effectiveness of your cache settings using [Cache Analytics](/cache/performance-review/cache-analytics/) and update your configuration according to our [Cache performance guide](/cache/performance-review/cache-performance/).
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+# Preparing for surges or spikes in web traffic
+
+## Use Cloudflare Cache features to optimize caching
+
+By default, Cloudflare [caches static content](/cache/concepts/default-cache-behavior/) such as images, CSS and JavaScript; however, you can extend Cloudflare caching to work with HTML by creating custom [Cache Rules](/cache/how-to/cache-rules/).
+
+### Cache more requests
+
+1. Log in to your Cloudflare dashboard.
```
Comment (marciocloudflare) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login), and select your account and domain.
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+2. Go to **Cache** > **Cache Rules** and select **Create Rule**.
+
+3. For `When incoming requests match…`, enter either your entire website or a specific path on your application, based on the **Hostname** or **URI Path**. Refer to the [available fields here](/cache/how-to/cache-rules/settings/#fields)
+
+4. For `Cache eligibility`, define how these requests should be cached and for how long. Refer to [available cache eligibility settings here](/cache/how-to/cache-rules/settings/#eligible-for-cache-settings)
+
+5. You can then monitor the effectiveness of your cache settings using [Cache Analytics](/cache/performance-review/cache-analytics/) and update according to our [Cache performance guide](/cache/performance-review/cache-performance/).
+
+Customers with Business and Enterprise domains have additional Page Rules settings that can be combined to selectively cache HTML content based on whether the page contains dynamic information (such as credentialed information), see our [Cache HTML selectively article here](/cache/troubleshooting/customize-caching/#cache-html-selectively-business-and-enterprise-domains)
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
Customers with Business and Enterprise domains have additional Page Rules settings that can be combined to selectively cache HTML content based on whether the page contains dynamic information (such as credentialed information). For more information, refer to [Cache HTML selectively](/cache/troubleshooting/customize-caching/#cache-html-selectively-business-and-enterprise-domains).
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md:23]:
```suggestion

```
Nit: Remove extra empy line.

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+
+### Advanced cache optimizations
+
+- [Custom Cache Keys](/cache/how-to/cache-keys/) allows you to precisely set the cacheability setting for any resource.
+
+- [Origin Cache Control](/cache/concepts/cache-control/) can be used to let the `Cache-Control` headers tell Cloudflare how to handle content from the origin.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
- [Origin Cache Control](/cache/concepts/cache-control/) can be used to let the `Cache-Control` headers tell Cloudflare how to handle content from the origin server. 
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+Customers with Business and Enterprise domains have additional Page Rules settings that can be combined to selectively cache HTML content based on whether the page contains dynamic information (such as credentialed information), see our [Cache HTML selectively article here](/cache/troubleshooting/customize-caching/#cache-html-selectively-business-and-enterprise-domains)
+
+
+### Advanced cache optimizations
+
+- [Custom Cache Keys](/cache/how-to/cache-keys/) allows you to precisely set the cacheability setting for any resource.
+
+- [Origin Cache Control](/cache/concepts/cache-control/) can be used to let the `Cache-Control` headers tell Cloudflare how to handle content from the origin. 
+
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
```
Nit: Remove extra empty line.

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 11]
```diff
@@ -0,0 +1,115 @@
...
+
+### Use Tiered Cache
+
+[Tiered Cache](/cache/how-to/tiered-cache/) uses the size of Cloudflare’s network to reduce requests to customer origins by dramatically increasing cache hit ratios.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
[Tiered Cache](/cache/how-to/tiered-cache/) uses the size of Cloudflare's network to reduce requests to customer origin servers by dramatically increasing cache hit ratios.
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 12]
```diff
@@ -0,0 +1,115 @@
...
+
+It works by dividing Cloudflare’s data centers into a hierarchy of lower-tiers and upper-tiers. If content is not cached in lower-tier data centers (generally the ones closest to a visitor), the lower-tier requests an upper-tier for the content. If the upper-tier does not have the content, only the upper-tier will initiate a request to the origin. This practice improves bandwidth efficiency by limiting the number of Cloudflare data centers that can ask the origin for content.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
It works by dividing Cloudflare's data centers into a hierarchy of lower-tiers and upper-tiers. If content is not cached in lower-tier data centers (generally the ones closest to a visitor), the lower-tier requests an upper-tier for the content. If the upper-tier does not have the content, only the upper-tier will initiate a request to the origin. This practice improves bandwidth efficiency by limiting the number of Cloudflare data centers that can ask the origin for content.
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 13]
```diff
@@ -0,0 +1,115 @@
...
+
+Refer to [Enable Tiered Cache](/cache/how-to/tiered-cache/#enable-tiered-cache) to get started.
+
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md:39]:
```suggestion
```
Nit: Remove extra empty line.

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
...
+## Use Cloudflare IP addresses to your advantage
+
+Take action to prevent attacks to your application during peak season by configuring your firewall to only accept traffic from Cloudflare IP addresses. By only allowing [Cloudflare IPs](https://www.cloudflare.com/ips), you can prevent attackers from bypassing Cloudflare and sending requests directly to your origin.
+
+Refer to [Cloudflare IP addresses](/fundamentals/concepts/cloudflare-ip-addresses/) for more information.
+
+## Monitor traffic in your Cloudflare Dashboard
+
+You can use the Cloudflare Dashboard to closely monitor the traffic on your domain and fine-tune your cache and security settings accordingly.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
## Monitor traffic in your Cloudflare dashboard

You can use the Cloudflare dashboard to closely monitor the traffic on your domain and fine-tune your cache and security settings accordingly.
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+
+### Zone and Account analytics
+
+The [Cloudflare zone analytics](/analytics/account-and-zone-analytics/zone-analytics/) is a major component of the overall Cloudflare Analytics product line.  Specifically, this app gives you access to a wide range of metrics, collected at the website or domain level.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
[Cloudflare zone analytics](/analytics/account-and-zone-analytics/zone-analytics/) gives you access to a wide range of metrics, collected at the website or domain level.
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+### Cache Analytics
+
+You can use [Cache Analytics](/cache/performance-review/cache-analytics/) to improve site performance or reduce origin web server traffic. 
+Cache Analytics helps determine if resources are missing from cache, expired, or ineligible for caching. 
+
+## Best Practices working with Cloudflare Support
+
+Before the high traffic event occurs, you must [open a Support ticket](/support/contacting-cloudflare-support/) and provide the information below.
+
+**For WAF/CDN customers**
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]: This WAF/CDN section is also covered in the DDoS section, not just the section "For Magic Transit and Spectrum customers". Should we add a link, or refactor the content to have the info in a single source (maybe in a partial)?

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
+---
+pcx_content_type: how-to
+title: Preparing for surges or spikes in web traffic
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
title: Prepare for surges or spikes in web traffic
```

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+
+## Use Cloudflare Cache features to optimize caching
+
+By default, Cloudflare [caches static content](/cache/concepts/default-cache-behavior/) such as images, CSS and JavaScript; however, you can extend Cloudflare caching to work with HTML by creating custom [Cache Rules](/cache/how-to/cache-rules/).
+
+### Cache more requests
+
+1. Log in to your Cloudflare dashboard.
+
+2. Go to **Cache** > **Cache Rules** and select **Create Rule**.
```
Comment (pedrosousa) [inline content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]:
```suggestion
2. Go to **Caching** > **Cache Rules** and select **Create rule**.
```
The UI has changed... We should probably review them to make sure they still work.

**Review by pedrosousa — APPROVED** [review]

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md (hunk 18 above)
Comment (jason-cf) [reply content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]: double checking

File: content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md (hunk 16 above)
Comment (jason-cf) [reply content/fundamentals/basic-tasks/preparing-for-surges-or-spikes-in-web-traffic.md]: I'm going to create a partial for both documents.

**Pull request merged**